transports/notifications/index.js

```diff
@@ -0,0 +1,121 @@
+/**
```

Comment:
There's some inconsistent spacing in this file, please use 4 spaces instead of tabs (or at least I think other files are doing so).